src/drive/components/IntentButton.jsx

```diff
@@ -0,0 +1,68 @@
+import styles from '../styles/intentbutton'
+
+/* global cozy */
+import React from 'react'
+import classNames from 'classnames'
+
+import Modal from 'cozy-ui/react/Modal'
+
+class IntentButton extends React.Component {
```

Comment:
This component seems to be very generic (not specific to an intent) and it could be great to have it directly in cozy-ui 🎉